iceoryx2-pal/posix/src/windows/signal.rs

```diff
@@ -201,3 +201,8 @@ pub unsafe fn sigemptyset(set: *mut sigset_t) -> int {
 pub unsafe fn sigpending(set: *mut sigset_t) -> int {
     unimplemented!("Windows does not support the sigset_t.");
 }
+
+pub unsafe fn abort() {
+    // TODO
+    unimplemented!("abort")
```

Comment:
The funny thing about this is, that the `unimplemented!` call will emit an `abort` signal which states that `abort` is not implemented ;)